examples/demo_table_comparisons.ipynb

```diff
@@ -0,0 +1,391 @@
+{
```

Comment:
<p>if you could provide pdf screenshot containing the answer, that will be better. I can help add it after you merge</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/run-llama/llama_parse/pull/55/discussion/#comment-1509649207'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>